KeyGuide/tests/KeyGuide-specs.js

```diff
@@ -1,111 +1,120 @@
-import {mount} from 'enzyme';
 import {FloatingLayerDecorator} from '@enact/ui/FloatingLayer';
+import '@testing-library/jest-dom';
+import {render} from '@testing-library/react';
 
 import KeyGuide from '../KeyGuide';
 
 const FloatingLayerController = FloatingLayerDecorator('div');
 
 describe('KeyGuide Specs', () => {
 	test('should not error with undefined children', () => {
-		const subject = mount(
+		const {queryAllByRole} = render(
 			<FloatingLayerController>
-				<KeyGuide open />
+				<KeyGuide data-testid="emptyKeyGuide" open />
```

Comment:
You are not using `data-testid` in this test. Is it necessary?

Reply:
It is not, will delete.